core/src/test/scala/unit/kafka/security/authorizer/AclAuthorizerTest.scala

```diff
@@ -714,7 +716,7 @@ class AclAuthorizerTest extends QuorumTestHarness with BaseAuthorizerTest {
 
   @Test
   def testThrowsOnAddPrefixedAclIfInterBrokerProtocolVersionTooLow(): Unit = {
-    givenAuthorizerWithProtocolVersion(Option(KAFKA_2_0_IV0))
+    givenAuthorizerWithProtocolVersion(Option(MetadataVersion.IBP_2_0_IV0))
```

Comment:
We can import `MetadataVersion.IBP_2_0_IV0` so that you only need to type `IBP_2_0_IV0` here